x-pack/plugins/synthetics/common/constants/ui.ts

```diff
@@ -14,7 +14,10 @@ export const MONITOR_EDIT_ROUTE = '/edit-monitor/:monitorId';
 export const MONITOR_MANAGEMENT_ROUTE = '/manage-monitors';
 
 export const OVERVIEW_ROUTE = '/';
-export const GETTING_STARTED_ROUTE = '/manage-monitors/getting-started';
+
+export const MONITORS_ROUTE = '/monitors';
+
+export const GETTING_STARTED_ROUTE = '/monitors/getting-started';
```

Comment:
i will suggest that we start separating constants as well between new and old app. Can be done in a follow up through.